front_end/entrypoints/inspector_main/InspectorMain.ts

```diff
@@ -15,6 +15,9 @@ import * as UI from '../../ui/legacy/legacy.js';
 
 import nodeIconStyles from './nodeIcon.css.js';
 
+const MS_BETWEEN_ROUNDTRIP_MEASUREMENTS = 3000;
```

Comment:
Curious if this should be more frequent, as (IIUC) this will create a very coarse, 3s incremented, metric? What about 1s/500ms?